src/ixdat/plotters/ec_plotter.py

```diff
@@ -12,18 +13,24 @@ def __init__(self, measurement=None):
         """Initiate the ECPlotter with its default Meausurement to plot"""
         self.measurement = measurement
 
+    @deprecate("0.1", "Use `v_name` instead.", "0.3", kwarg_name="V_str")
+    @deprecate("0.1", "Use `j_name` instead.", "0.3", kwarg_name="J_str")
```

Comment:
Here BTW we might excuse ourselves from writing the kwarg names out to keep it on one line.